start-site/src/main/resources/application.yml

```diff
@@ -83,11 +83,14 @@ initializr:
           - compatibilityRange: "[2.4.0.M2,2.4.0-M3]"
             version: 2020.0.0-M4
             repositories: spring-milestones
-          - compatibilityRange: "[2.4.0.M4,2.4.x-SNAPSHOT)"
+          - compatibilityRange: "2.4.0.M4"
```

Comment:
This doesn't look quite right. `2.4.0.M4` means `2.4.0.M4` and later. As such it overlaps with `[2.4.0,2.5.0-M1)` below.

Reply:
ah, I thought it means `2.4.0.M4` only...